### PR TITLE
Update Readme and Makefile for using MySQL

### PR DESCRIPTION
**Pull request by g0djan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,24 +187,64 @@ Delta computation is based on ModTime of file system and LSN number of pages in
...
-To configure network upload rate limit during ```backup-push``` in bytes per second.
+* `WALG_MYSQL_BINLOG_SRC`
+
+To configure directory with binlogs for ```binlog-push```.
+
+* `WALG_MYSQL_BINLOG_END_TS`
+
+To set time [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) for recovery point.
+
+* `WALG_MYSQL_SSL_CA`
```
Comment (mialinx) [inline README.md]: Do we need any special options in DSN in order for this SSL config to be used ?

File: README.md [hunk 2]
```diff
@@ -230,52 +287,69 @@ When uploading WAL archives to S3, the user should pass in the absolute path to
 wal-g wal-push /path/to/archive
 ```
 
-* ``backup-list``
+### MySQL
 
-Lists names and creation time of available backups.
+* ``stream-fetch``
```
Comment (mialinx) [inline README.md]: stream-fetch is not mysql-specific tool. why it's in mysql section ?

File: README.md [hunk 3]
```diff
@@ -230,52 +287,69 @@ When uploading WAL archives to S3, the user should pass in the absolute path to
...
-* ``delete``
-
-Is used to delete backups and WALs before them. By default ``delete`` will perform dry run. If you want to execute deletion you have to add ``--confirm`` flag at the end of the command.
-
-``delete`` can operate in two modes: ``retain`` and ``before``.
-
-``retain`` [FULL|FIND_FULL] %number%
+When fetching backup's stream, the user should pass in the name of the backup. It returns an encrypted data stream to stdout, you should pass it to a backup tool that you used to create this backup.
+```
+wal-g stream-fetch example-backup | some_backup_tool use_backup
```
Comment (mialinx) [inline README.md]: what is some_mysql_tool ? mysql client ?

File: README.md [hunk 4]
```diff
@@ -230,52 +287,69 @@ When uploading WAL archives to S3, the user should pass in the absolute path to
...
 
-``before`` [FIND_FULL] %name%
+* ``stream-push``
 
-if FIND_FULL is specified WAL-G will calculate minimum backup needed to keep all deltas alive. If FIND_FULL is not specified and call can produce orphaned deltas - call will fail with the list.
+Command for compressing, encrypting and sending backup from stream to storage.
 
-``retain 5`` will fail if 5th is delta
+```
+some_backup_tool make_backup | wal-g stream-push
```
Comment (mialinx) [inline README.md]: what is some_backup_tool ? mysqldump ?

File: README.md (hunk 1 above)
Comment (g0djan) [reply README.md]: @x4m

File: README.md (hunk 2 above)
Comment (g0djan) [reply README.md]: Yep, but it still needs refactoring for making it common. Is it okay if we place it here right now, and when I make it common move it to a common part of README?

File: README.md (hunk 1 above)
Comment (g0djan) [reply README.md]: Can you give an example of suggestion options? Because we don't know anything about it.

File: README.md (hunk 2 above)
Comment (mialinx) [reply README.md]: ok

File: README.md (hunk 1 above)
Comment (mialinx) [reply README.md]: WALG_MYSQL_SSL_CA  implicitly add tls=custom option to DSN.  So you should not specify any  tls option in DSN by yourself

**Comment by mialinx** [comment]
LGTM

**Pull request merged**